helm/README.md

```diff
@@ -14,12 +14,7 @@ export NGC_CLI_API_KEY="key from ngc"
 If you have not set up NGC, see the [NGC Setup](https://ngc.nvidia.com/setup) topic.
 
 [comment]: <> (TODO: update the repo with th real location)
-Add the helm repo with:
-
-```bash
-helm repo add nim-demo "https://nim-deploy.github.io/nvidia/nim-llm"
-```
-If using pre-release NIM, change the repo URL accordingly.
+Clone this repo to have this heml chart locally. Make sure you are in `nemo-deploy/helm`.
```

Comment:
heml -> helm

Reply:
Can we try to use impersonal language in the readmes to conform to style guides?

Rather than "make sure you", "Commands must be run from the `nemo-deploy/hellm` repo. But is nemo-deploy corrrect or should that ne nim-deploy/helm?